pkgs/development/python-modules/pyheos/default.nix

```diff
@@ -28,10 +29,17 @@ buildPythonPackage rec {
     syrupy
   ];
 
-  disabledTests = [
-    # accesses network
-    "test_connect_timeout"
-  ];
+  disabledTests =
+    [
+      # accesses network
+      "test_connect_timeout"
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      # OSError: could not bind on any address out of [('127.0.0.2', 1255)]
+      "test_failover"
+    ];
```

Comment:
Wait, isn't __darwinAllowLocalNetworking supposed to fix exactly this?

Reply:
It fixes most tests but maybe 127.0.0.2, unlike 127.0.0.1, can't be bound to on macOS.